app/forms/sufia/forms/batch_edit_form.rb

```diff
@@ -1,9 +1,46 @@
 module Sufia
   module Forms
-    class BatchEditForm < CurationConcerns::Forms::FileSetEditForm
-      self.terms = [:resource_type, :title, :creator, :contributor, :description,
-                    :keyword, :rights, :publisher, :date_created, :subject, :language,
+    class BatchEditForm < CurationConcerns::Forms::WorkForm
+      self.terms = [:creator, :contributor, :description,
```

Comment:
@mjgiarlo title is on a per item basis along with resource_type.  They were being removed in the controller before here: https://github.com/projecthydra/sufia/pull/2031/files#diff-b576b2d8a3e70510dfc0858617ee80ecL15

I do always get confused on date_created though.  Is that just a text field?  Maybe I should put that one back...